packages/rocketchat-slashcommands-unflip/unflip.js

```diff
@@ -0,0 +1,20 @@
+/*
+* Unflip is a named function that will replace /unflip commands
+* @param {Object} message - The message object
+*/
+
+
+function Unflip(command, params, item) {
+	if (command === 'Unflip') {
+		var msg;
+
+		msg = item;
+		msg.msg = params + ' ¯(╯°□°）╯︵ ┻━┻';
+		Meteor.call('sendMessage', msg);
+	}
+}
+
+RocketChat.slashCommands.add('Unflip', Shrug, {
```

Comment:
Also lowercase here, for consistancy.  As well as use Unflip instead of Shrug here.  Otherwise typing /Unflip will result in it showing the shrug.